services/core-api/app/api/mines/permits/permit_amendment/models/permit_amendment.py

```diff
@@ -210,6 +210,12 @@ def find_by_permit_id(cls, _id):
         return cls.query.filter_by(permit_id=_id).filter_by(deleted_ind=False).filter(
             cls.permit_amendment_status_code != 'DFT').all()
 
+    @classmethod
+    def find_last_by_permit_id(cls, _id):
+        return cls.query.filter_by(permit_id=_id).filter_by(deleted_ind=False).filter(
```

Comment:
can you rename to mention what is being found? 

ie found_last_amendment_by_permit_id ?

Reply:
Thank you. Modifying and pushing this change.